api/core/event/index.js

```diff
@@ -2,5 +2,6 @@
 module.exports.create = require('./event.create.js');
 module.exports.command = require('./event.command.js');
 module.exports.get = require('./event.get.js');
+module.exports.getByEventtype = require('./event.getByEventtype.js');
```

Comment:
Juste one feedback, can you call your function `getByEventType` ? camelcase is more beautiful :)

I know it's not the case everywhere in Gladys, but the goal is to be camelcase everywhere in the JS code